### ISSUE TITLE
Plot runtime of map_fn, vectorized_map, and a true vectorized implementation for RandomErasing

### ISSUE THREAD

**Issue by LukeWood** [keras-team/keras-cv#160]
This will help us determine which flows we need to support/how important each improvement is on the previous.

**Comment by bhack** [comment]
Do we need to add also XLA `jit_compile`? As nobody clarified on https://github.com/keras-team/keras-cv/pull/146 if it was or not orthogonal to   `vectorized_map` and to a true vectorized implementation.

**Comment by bhack** [comment]
For `jit_compile` we could check if we want to suggest users to enable with `tf.config.optimizer.set_jit` like in `https://www.tensorflow.org/xla/tutorials/autoclustering_xla` instead of enforcing ourself in the `@tf.function` decorator.

We have already some layers performance tests in Keras with `tf.function` vs XLA `jit_compile` matrix in:

https://github.com/keras-team/keras/blob/master/keras/benchmarks/layer_benchmarks/layer_benchmarks_test.py

**Comment by LukeWood** [comment]
The purpose of this experiment is to gather performance numbers so we can make a decision.  I'll be doing benchmarking as thoroughly as possible to get this data.

**Comment by bhack** [comment]
Yes I just meant if we want to collect also some performance with `jit_compile` in the performance matrix.

**Comment by chjort** [comment]
@LukeWood 
I have made some performance improvements to `keras_cv.utils.fill_utils` in PR #159. So I would recommend benchmarking with these improvements. I could include these performance improvements in its own PR without the GridMask changes if you would like.

**Comment by LukeWood** [comment]
the goal is to determine how good `vectorized_map` is more than anything, so I think the benchmark is fine.
